index.js

```diff
@@ -18,7 +37,25 @@ const manageRecipes = async () => {
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
 
-    // Run your code here, after you have insured that the connection was made
+    //Create one Recipe
+    let titleRecipe = await Recipe.create(newRecipe);
+    console.log("Nova Receita -->" + titleRecipe.title);
+
+    //Create all Recipes
+    await Recipe.insertMany(data);
```

Comment:
If you wanted to `console.log` only the titles of the recipes, you needed to loop over your `allRecipes` variable that is storing an array with your recipes objects.
Something like this:
```
 let allRecipes = await Recipe.insertMany(data);
// console log to see the titles of the inserted recipes
    for (let i = 0; i < allRecipes.length; i++) {
      console.log(allRecipes[i].title);
    }
```